deploy/crds/csi.ibm.com_v1_ibmblockcsi_cr.yaml

```diff
@@ -47,24 +51,24 @@ spec:
 
   sidecars:
   - name: csi-node-driver-registrar
-    repository: quay.io/k8scsi/csi-node-driver-registrar
-    tag: "v1.2.0"
```

Comment:
what is the meaning of switching from quay?

Reply:
it means that the images will be pulled from a different registry (quay.io is [deprecated](https://github.com/kubernetes-csi/csi-release-tools/issues/86))